source/hardware/controllers/native_instruments_traktor_kontrol_s4_mk3.rst

```diff
@@ -405,12 +425,24 @@ Define the tension of the jogwheel                               `TightnessFacto
 Define how much force can the jogwheel use                       `MaxWheelForce`                             25000          10000..30000 (Recommended, can go up to 60000) Define how much resistance can the wheel use when its rotation is held
 ================================================================ =========================================== ============== ============================================== ==========================================================================================================================================================
 
+Changing settings
+~~~~~~~~~~~~~~~~~
+
+Mixxx 2.4
+^^^^^^^^^
 
-Making these changes is still a little awkward and we will be making controller preferences easier to change in the future.
+Making these changes is still a little awkward, this will be much easier to do starting in Mixxx 2.5.
 For now you’ll have to make a small change to the mapping script file. Don’t worry, the actual edit only involves replacing a single word in a text file.
 
 1. Open Mixxx Preferences and select the Kontrol S4 in the side list.
 2. Select :file:`Traktor-Kontrol-S4-MK3-hid-scripts.js`.
 3. Click :guilabel:`Open Selected File`.
 4. Either the file should open in an editor, or you should see a file browser window with that file selected. If you see a file browser, right click the file and select an option to edit it.
 5. Starting from the block entitled `USER CONFIGURABLE SETTINGS`, there should be configurable options.
+
+
+
+Mixxx 2.5+
+^^^^^^^^^^
+
+Settings can be edited in the preference windows, under :guilabel:`Preferences` > :guilabel:`Controllers` > :guilabel:`YourController`.
```

Comment:
Isn't it "Traktor S4 Mk3" on all platforms?

Reply:
In my case, it seems to be prefixed with a string: `Traktor Kontrol S4 MK3 AF08_3`. That feels too short to be a serial number, but perhaps this is the same for everyone? I'll ask on Zulip if one of our user on Mac can confirm he has the same.

Reply:
Looks like the user is currently unavailable - are you happy with the current suggestion?